backend/src/game_objects/content.rs

```diff
@@ -0,0 +1,26 @@
+use serde::Serialize;
+
+#[derive(Clone, Serialize)]
+pub enum WallKind {
+    Normal,
```

Comment:
I think the more appropriate world for the default wall kind would be "Default" or "Regular".
"Normal" suggests that there would be some abnormal, eerie, ill-boding walls, which is not exactly our case (yet).

Reply:
sure